### PR TITLE
Importing pycon-fr-2019 from dl.afpy.org, ponyconf, and youtube

### PR DESCRIPTION
**Pull request by JulienPalard**

### CODE REVIEW

File: pycon-fr-2019/A-Dive-Into-Brain-Waves.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "language": "fra",
+    "recorded": "2019-11-02",
+    "speakers": [
+        "Utkarsh Shukla"
+    ],
+    "videos": [
+        {
+            "type": "mp4",
+            "size": 1957845188,
+            "url": "https://dl.afpy.org/pycon-fr-19/Robin%20Huart%20-%20Tests%20-%20la%20jungle%20des%20frameworks/Robin%20Huart%20-%20Tests%20-%20la%20jungle%20des%20frameworks.mp4"
```
Comment (pilou-) [inline pycon-fr-2019/A-Dive-Into-Brain-Waves.json]: It looks like this video doesn't match the title/description.

Comment (JulienPalard) [reply pycon-fr-2019/A-Dive-Into-Brain-Waves.json]: Yes I know there's a few of them, I matched between ponyconf and files using levenstein distance between titles, and I spotted it mis-matching a few of them (it typically don't work well when one of the side has been truncated, truncation being a "big edit" from the Levenstein point of view).

Comment (JulienPalard) [reply pycon-fr-2019/A-Dive-Into-Brain-Waves.json]: I'll proofread the thing, consider it WIP. It still miss the youtube videos also (do you have the links to youtube uploads?)

**Comment by redapple** [comment]
@JulienPalard , if you need help with anything, let me know.
YouTube videos are not mandatory (I can't find any playlist on YouTube by the way)

**Comment by JulienPalard** [comment]
Someone gave me the youtube videos (still unlisted), so I'll can add them. I think I can script this with a few lines of Python :)

**Comment by JulienPalard** [comment]
I think I properly reconciliated Ponyconf (title / descriptions / speakers), dl.afpy.org (video files) and youtube.

I have two videos that miss a youtube upload but it can be fixed later:

    videos/Les-decorateurs:-Kezako?.json
    videos/LSTM-Variational-AutoEncoders-for-Network-Signal-Anomaly-Detection.json

**Comment by JulienPalard** [comment]
(I like how I commit "FIX: Markdown" while fixing reStructuredText :P)

**Comment by pilou-** [comment]
> I have two videos that miss a youtube upload but it can be fixed later:
> 
> ```
> videos/Les-decorateurs:-Kezako?.json
> videos/LSTM-Variational-AutoEncoders-for-Network-Signal-Anomaly-Detection.json
> ```

First one was still private: https://www.youtube.com/watch?v=JD1eknK3hxo
Last one was public but not part of the playlist: https://www.youtube.com/watch?v=L7m8wIRA11A

File: pycon-fr-2019/videos/Les-decorateurs:-Kezako?.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "speakers": [
+    "Alexis Benoist"
+  ],
+  "title": "Les d\u00e9corateurs\u202f: K\u00e9zako\u202f?",
+  "videos": [
+    {
+      "size": 111036965,
+      "type": "webm",
+      "url": "https://dl.afpy.org/pycon-fr-19/Alexis%20Benoist%20-%20Les%20decorateurs%20-%20Kezako/Alexis%20Benoist%20-%20Les%20decorateurs%20-%20Kezako.webm"
+    }
```
Comment (pilou-) [inline pycon-fr-2019/videos/Les-decorateurs:-Kezako?.json]:
```suggestion
    },
    {
        "type": "youtube",
        "url": "https://www.youtube.com/watch?v=JD1eknK3hxo"
    }
```

File: pycon-fr-2019/videos/LSTM-Variational-AutoEncoders-for-Network-Signal-Anomaly-Detection.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    {
+      "size": 1564088776,
+      "type": "mp4",
+      "url": "https://dl.afpy.org/pycon-fr-19/Facundo%20Calcagno%20-%20LSTM%20Variational%20AutoEncoders%20for%20Network%20Signal%20Anomaly%20Detection/Facundo%20Calcagno%20-%20LSTM%20Variational%20AutoEncoders%20for%20Network%20Signal%20Anomaly%20Detection.mp4"
+    },
+    {
+      "size": 220119851,
+      "type": "webm",
+      "url": "https://dl.afpy.org/pycon-fr-19/Facundo%20Calcagno%20-%20LSTM%20Variational%20AutoEncoders%20for%20Network%20Signal%20Anomaly%20Detection/Facundo%20Calcagno%20-%20LSTM%20Variational%20AutoEncoders%20for%20Network%20Signal%20Anomaly%20Detection.webm"
+    }
```
Comment (pilou-) [inline pycon-fr-2019/videos/LSTM-Variational-AutoEncoders-for-Network-Signal-Anomaly-Detection.json]:
```suggestion
    },
    {
        "type": "youtube",
        "url": "https://www.youtube.com/watch?v=L7m8wIRA11A"
    }
```

**Comment by JulienPalard** [comment]
> It looks like this one is missing: https://www.youtube.com/watch?v=v_EZTo45FtE

Nice catch!

**Comment by JulienPalard** [comment]
@redapple looks OK to me (but beware it's my first PR on pyvideo).

**Comment by JulienPalard** [comment]
(rebased and squashed)

**Comment by redapple** [comment]
Thanks so much @JulienPalard & @pilou- 
I just added the thumbnails and slugified filenames.

**Comment by JulienPalard** [comment]
Thanks @redapple for this final cleaning and merging :)

**Comment by JulienPalard** [comment]
Spotted an issue related to this import: https://github.com/pyvideo/pyvideo/issues/94

**Comment by redapple** [comment]
@JulienPalard the issue with tabs is now fixed.

I noticed that https://www.youtube.com/watch?v=UV0k12sGues is not available anymore.
Do you know if there's an alternative? or should be remove the YouTube link?

**Comment by pilou-** [comment]
> I noticed that https://www.youtube.com/watch?v=UV0k12sGues is not available anymore.
> Do you know if there's an alternative? or should we remove the YouTube link?

There was an issue with this video, new URL is https://www.youtube.com/watch?v=UpmZIble3gQ.

**Pull request merged**